components/docusign/docusign.app.js

```diff
@@ -0,0 +1,129 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "docusign",
+  propDefinitions: {
+    account: {
+      type: "string",
+      label: "Account",
+      description: "",
+      async options() {
+        const { accounts } = await this.getUserInfo();
+        return accounts.map((account) => {
+          return {
+            label: account.account_name,
+            value: account.account_id,
+          };
+        });
+      },
+    },
+    template: {
+      type: "string",
+      label: "Template",
+      description: "",
+      async options({ account }) {
+        const baseUri = await this.getBaseUri(account);
+        const { envelopeTemplates } = await this.listTemplates(
+          baseUri,
+          account,
+        );
+        return envelopeTemplates.map((template) => {
+          return {
+            label: template.name,
+            value: template.templateId,
+          };
+        });
+      },
+    },
+    role: {
+      type: "string",
+      label: "Recipient Role",
+      description: "",
```

Comment:
Can you add the same description Zapier has for their action?

Reply:
I'd also move this last prop to show up last in the list (also like Zapier)

Reply:
Updated